src/cluster/purging.rst

```diff
@@ -0,0 +1,173 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cluster/purging:
+
+===============
+Clustered Purge
+===============
+The primary purpose of clustered purge is to clean databases that have multiple
+deleted tombstones or single documents that contain large numbers of conflicts.
+But it can also be used to purge any document (deleted or non-deleted) with any
+number of revisions.
+
+Clustered Purge is designed to maintain eventual consistency and prevent
+unnecessary invalidation of secondary indexes. For this, every database keeps
+track of a certain number of historical purges requested in the database, as
+well as its current ``purge_seq``. Internal replications and secondary indexes
+process database's purges and periodically update their corresponding purge
+checkpoint documents to report ``purge_seq`` processed by them. To ensure
+eventual consistency, the database will remove store historical purge requests
```

Comment:
remove the word "store":  "the database will remove historical purge requests..."

Or maybe it should be "stored"?  "the database will remove stored historical purge requests..."